tests/layer_tests.py

```diff
@@ -237,3 +237,87 @@ def test_regex_split(test_string, expected, layer):
     compiled_model = create_splitting_model(layer)
     res_ov = compiled_model([test_string])[0]
     assert (res_ov == expected).all()
+
+
+################################################
+######## Test RaggedToDense Operation ##########
+################################################
+
+def infer_ragged_to_dense(test_inputs: List[List[int]], padding: str) -> List[np.array]:
+    begins = op.Parameter(Type.i32, PartialShape(["?"]))
+    ends = op.Parameter(Type.i32, PartialShape(["?"]))
+    data = op.Parameter(Type.i32, PartialShape(["?"]))
+    vaulue = op.Parameter(Type.i32, PartialShape(["?"]))
+    padding_size = op.Parameter(Type.i32, PartialShape([]))
+
+    input_params = [begins, ends, data, vaulue, padding_size]
+    assert padding in ["right", "left"]
+    pad_right = True if padding == "right" else False
+    combine_segments = _get_factory().create("RaggedToDense", input_params, {"pad_right": pad_right}).outputs()
+
+    ragged_to_dense_model = Model(combine_segments, input_params, "ragged_to_dense")
+    # ov.save_model(ragged_to_dense_model, 'ragged_to_dense.xml')
+
+    compiled_model = core.compile_model(ragged_to_dense_model)
+    return compiled_model([np.array(inp, dtype=np.int32) for inp in test_inputs])
+
+
+@pytest.mark.parametrize(
+    "test_inputs, padding, expected",
+    [   # beging, ends, data, padding_size, value
+        (([0, 3], [3, 8] , [10, 20, 100, 30, 40, 50, 200, 300], 10, 42), 
```

Comment:
Using dicts to organize `test_inputs` is better for readability here and in the next test, too.
```python
{
    "begins": [0, 3],
    "ends": [3, 8],
    "data": [10, 20, 100, 30, 40, 50, 200, 300],
    ...
}
```

Reply:
done